demoscene/utils/groklinks.py

```diff
@@ -1531,12 +1531,70 @@ def __unicode__(self):
         return u"https://%s.bandcamp.com/track/%s" % (domain, name)
 
 
+class Defactro2Entry(BaseUrl): # for use as an abstract superclass
```

Comment:
Typo here - 'Defactro' should be 'Defacto'

Reply:
A “defactro” is a type of intro that tells it like it is!